ecf/scripts/gfs/wave/prep/jgfs_wave_prep.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:30:00
-#PBS -l place=vscatter,select=1:ncpus=128
+#PBS -l place=vscatter,select=1:mpiprocs=60:ompthreads=1:ncpus=60
 #PBS -l place=excl
```

Comment:
Does this job still need to be exclusive?

Reply:
same as the gdas wave prep job.